e2e/js_image_docker/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "image",
     "dependencies": {
-        "chalk": "^5.0.0"
+        "chalk": "^4.0.0"
```

Comment:
This can land on main? I'll put up a PR there

Reply:
Yes it can, and should